apps/src/templates/sectionProgress/progressTables/ProgressTableDetailCell.jsx

```diff
@@ -0,0 +1,203 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {
+  levelTypeWithoutStatus,
+  studentLevelProgressType
+} from '@cdo/apps/templates/progress/progressTypes';
+import ProgressTableLevelBubble from './ProgressTableLevelBubble';
+import * as progressStyles from '@cdo/apps/templates/progress/progressStyles';
+import color from '@cdo/apps/util/color';
+import firehoseClient from '@cdo/apps/lib/util/firehose';
+import _ from 'lodash';
+import {LevelStatus} from '@cdo/apps/util/sharedConstants';
+
+const UNPLUGGED_BUBBLE_WIDTH = getUnpluggedWidth();
+
+const styles = {
+  container: {
+    ...progressStyles.flexBetween,
+    position: 'relative',
+    whiteSpace: 'nowrap'
+  },
+  background: {
+    height: 10,
+    backgroundColor: color.lighter_gray,
+    position: 'absolute',
+    left: 10,
+    right: 10
+  },
+  sublevelContainer: {
+    position: 'relative',
+    display: 'inline-block'
+  }
+};
+export default class ProgressTableDetailCell extends React.Component {
+  static widthForLevels(levels) {
```

Comment:
Where is this called from?

Reply:
`ProgressTableDetailView`, which needs to pass an array of explicit column widths into `ProgressTableContentView`. we have not yet opened a PR for either of those components though.

Reply:
Ok, I'll take your word for it.  :-)